changelogs/fragments/1962-zos_job_submit-log-generated.yml

```
@@ -0,0 +1,3 @@
+trivial:
```

I understand the initial thought of setting this as trivial since is just logging, but at the same time I think this is something we want users to know about, now they will have the ability to view the rendered template.

```suggestion
minor_changes:
```